examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,216 @@
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: 0'),
+      ),
+      findsOne,
+      reason: 'Expected element with text "Item: 0" in the CustomScrollView',
+    );
+  });
+
+  testWidgets('IconButton click extends existing SliverList', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.byType(SliverList),
+      ),
+      findsOne,
+      reason: 'Expected one, initial (bottom) SliverList in the CustomScrollView',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(SliverList),
+        matching: find.byType(Container),
+      ),
+      findsOne,
+      reason: 'Expected one, initial (bottom) Container in the SliverList',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: 1'),
+      ),
+      findsNothing,
+      reason: 'Expected no element with text "Item: 1" in the CustomScrollView',
+    );
+
+    // Tap the IconButton in the AppBar
+    final Finder iconButtonFinder = find.descendant(
+      of: find.byType(AppBar),
+      matching: find.byType(IconButton),
+    );
+    await tester.tap(iconButtonFinder);
+    await tester.pump();
+
+    expect(
+      find.descendant(
+        of: find.byType(SliverList),
+        matching: find.byType(Container),
+      ),
+      findsExactly(2),
+      reason: 'Expected additional Container in the SliverList after the IconButton click',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: 1'),
+      ),
+      findsOne,
+      reason: 'Expected element with text "Item: 1" in the CustomScrollView',
+    );
+  });
+  testWidgets('IconButton click and mouse scroll reveals additional SliverList', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: -1'),
+      ),
+      findsNothing,
+      reason: 'Expected no element with text "Item: -1" in the CustomScrollView',
+    );
+
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.text('Item: 1'),
+      ),
+      findsNothing,
+      reason: 'Expected no element with text "Item: 1" in the CustomScrollView',
+    );
+
+    // First check before we start
+    expect(
+      find.descendant(
+        of: find.byType(CustomScrollView),
+        matching: find.byType(SliverList),
+      ),
+      findsOne,
+      reason: 'Expected to have only one, initial (bottom) SliverList in the CustomScrollView',
+    );
+
+    // Second check before we start.
+    // Initially, mouse scroll event should do nothing.
+    // It should not reveal additional (top) SliverList 
```

Comment:
```suggestion
    // It should not reveal additional (top) SliverList
```

Apparently, [Linux Analyze](https://github.com/flutter/flutter/pull/152431/checks?check_run_id=28241547944) is upset about this 😒